File-Formats/XlsIO/Working-with-Template-Markers.md

```diff
@@ -497,7 +497,7 @@ By default, DataTable values will be filled in the worksheet as a string format.
 
  The following screenshot represents the input template which has a template marker.
 
-![Bind from DataTable Example](Working-with-Template-Markers_images/Working-with-Template_Markers_img3.jpeg)
+![Bind from DataTable Example](Working-with-Template-Markers_images/Bind_data_from_datatable_to_Excel_Spreadsheet_Template.jpeg)ss
```

Comment:
remove the unwanted text
![image](https://user-images.githubusercontent.com/61148953/182309636-f03c16b4-41de-466b-98e0-d5429ac37760.png)